pennylane/resource/resource.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
I don't think we need to update this if nothing has changed in this file.

Reply:
Removed from files that were not updated